demo/bottomup_demo.py

```diff
@@ -81,13 +85,15 @@ def parse_args():
     parser.add_argument(
         '--radius',
         type=int,
-        default=3,
+        default=6,
         help='Keypoint radius for visualization')
     parser.add_argument(
         '--thickness',
         type=int,
-        default=1,
+        default=3,
         help='Link thickness for visualization')
```

Comment:
Would these two values be excessively large for visualization using `PoseLocalVisualizer`, which is adopted for image inputs?